packages/expo-cli/src/commands/customize.ts

```diff
@@ -31,9 +31,38 @@ async function maybeWarnToCommitAsync(projectRoot: string) {
   }
 }
 
-const dependencyMap: { [key: string]: string[] } = {
-  'babel.config.js': ['babel-preset-expo'],
-  'webpack.config.js': ['@expo/webpack-config'],
+const dependencyMap: {
```

Comment:
Not certain how this works, so please correct me, but I wonder if this configuration should live in the same package as the templates itself. Right now if you edit the templates, you will also need to edit this file in `expo-cli`, which consumes the templates?